### ISSUE TITLE
Hi to old Secretislanders

### ISSUE THREAD

**Issue by andreditvirs** [EpicConOwO/Htc_Desire_626G_PLUS_twrp#1]
Hello @EpicConOwO still remember me XD I think you were online in github but you didnt online in discord, I think you are very busy business man now XD. I remember that you are developer too, I cant contact you by email or what because i didnt have way to conect with you. I hope you will able to be back to secretisland server discord when you know this message (issue) XD.

Regards,
AndreDitVirs
https://discord.gg/PeFsg7yT

**Comment by EpicConOwO** [comment]
OUH hi andrew!!, yes i remember sorry i actually kind of forgot my discord
tag ._. and i tried guessing but to no avail.  also no i was just bussy
with university stuff then i got side tracked by tensorflow and while
working on that i got side tracked even more trying to communicate java
with python (idk why but i just liked the idea at first) , currently im
making a name generator with java and python,  company names are just so
hard to find these days ;w; and yes we i can join secretisland lets connect
on discord or somewhere cause its gonna be hectic going back and forth with
email.

**Comment by andreditvirs** [comment]
> OUH hi andrew!!, yes i remember sorry i actually kind of forgot my discord tag ._. and i tried guessing but to no avail. also no i was just bussy with university stuff then i got side tracked by tensorflow and while working on that i got side tracked even more trying to communicate java with python (idk why but i just liked the idea at first) , currently im making a name generator with java and python, company names are just so hard to find these days ;w; and yes we i can join secretisland lets connect on discord or somewhere cause its gonna be hectic going back and forth with email.

Yeah @EpicConOwO XD I think you are a busy man now 👍 I just want to know how are you today. Good to know you are ok now XD. May be we can play together if you are ready and join our old discord server here : https://discord.gg/PeFsg7yT

Secretisland will always open the door for you :D see you there 
![image](https://user-images.githubusercontent.com/51712370/142200818-13e629b2-eb07-4ced-9f5a-bcbeda6069f5.png)

**Comment by EpicConOwO** [comment]
hey andrew sorry for late reply i was doing some iot projects , UH yes sure
but i will need uh bit of help- uh can you like please copy and paste the
old discord account username and tag that way i wont have redundant
accounts? and also yes im good how about you?

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by andreditvirs** [comment]
![image](https://user-images.githubusercontent.com/51712370/142942083-8d507879-e913-464c-a00e-5676730a768d.png)
Your account @EpicConOwO ? EpicCon#9719
